turbinia/output_manager.py

```diff
@@ -495,3 +496,51 @@ def copy_from(self, source_path):
       log.error(message)
       raise TurbiniaException(message)
     return destination_path
+
+  def copy_from_gcs(self, saved_paths):
+    """Copies output file from the managed location to the local output dir.
+
+    Args:
+      saved_paths (list): A list of saved paths. List includes GCS and local
+      output file paths.
+
+    Raises:
+      TurbiniaException: If file retrieval fails.
+    """
+    bucket = self.client.get_bucket(self.bucket)
+
+    for path in saved_paths:
+      if 'gs://' in path:
+        gcs_path = self._parse_gcs_path(path)[1]
+
+        try:
+          # Reconstruct the same file structure as GCS on the output dir
+          path_split = gcs_path.split('/')
+          directory = os.path.join(''.join(path_split[0:-1]))
```

Comment:
Why is the inner "".join() here?  I'm pretty sure you don't want that since it will combine all of the path segments with nothing between them (so the outer os.path.join() wouldn't have any effect).

Reply:
This is needed since os.path doesn't accept lists as argument. It's basically converting our path list to str for os.path to digest. See below for the error:
```    
directory = os.path.join(path_split[0:-1])
  File "/usr/lib/python3.8/posixpath.py", line 76, in join
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not list
```

Reply:
Ah, I didn't realize os.path.join didn't take a list.  You can expand/unpack the list into individual arguments with the `*` operator too (see below).   I think the inner join call is not doing quite what you'd expect though since it will join the part segments with nothing between them (e.g. `["home", "user", "dir"]` turns into `"homeuserdir"`).

```suggestion
          directory = os.path.join(*path_split[0:-1]))
```